packages/modelviewer.dev/src/docs-and-examples/sidebar.ts

```diff
@@ -208,7 +209,7 @@ function handlePageJump(entries: IntersectionObserverEntry[]) {
 /*
  * Update the table of contents based on how the page is viewed.
  */
-export function sidebarObserver(docsOrExamples: 'docs'|'examples') {
+export function sidebarObserver(docsOrExamples: string) {
```

Comment:
Maybe `docsOrExample` since we're choosing a specific one now?

Reply:
hmm, yeah, I think you're right, I'll update that in the next one